airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -681,14 +681,25 @@ def _build_cmd(variables: Dict, label_formatter: Callable, project_id: str) -> L
             "--runner=DataflowRunner",
             "--project={}".format(project_id),
         ]
-        if variables is not None:
-            for attr, value in variables.items():
-                if attr == 'labels':
-                    command += label_formatter(value)
-                elif value is None or value.__len__() < 1:
+        if variables is None:
+            return command
+
+        # The logic of this method should be compatible with Apachea Beam:
+        # https://github.com/apache/beam/blob/b56740f0e8cd80c2873412847d0b336837429fb9/sdks/python/
+        # apache_beam/options/pipeline_options.py#L230-L251
+        for attr, value in variables.items():
+            if attr == 'labels':
+                command += label_formatter(value)
+            elif value is None:
+                command.append("--" + attr)
+            elif isinstance(value, bool):
+                if value:  # pylint: disable=too-many-nested-blocks
```

Comment:
```suggestion
            elif isinstance(value, bool) and value:
```

Reply:
Here `value is True` will also work

Reply:
Thanks.  Did you know that you can already add suggestions on multiple lines?
https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/
Demo: 
https://github.com/apache/airflow/pull/8394#discussion_r410908722
Thanks to this, you can quickly edit files from Web UI.

Reply:
Yeah I know